src/d8/drupal.make.yml

```diff
@@ -0,0 +1,21 @@
+core: 8.x
+api: 2
+
+# Common attributes.
+defaults:
+  projects:
+    subdir: "contrib"
+
+projects:
+  drupal:
+    version: ~
+  features:
+    version: ~
+  field_group:
+    version: ~
+  panels:
+    version: ~
+  pathauto:
+    version: ~
+  token:
+    version: ~
```

Comment:
that's been done actually. if anything we might want to add some to it. I did weed out ones that weren't availiable like master, features_extra, and the aurora theme.